Scripts/NuGet/EdFi.Installer.WebApi/Install-EdFiOdsWebApi.psm1

```diff
@@ -390,12 +396,18 @@ function Invoke-TransformWebConfigAppSettings {
             $settings = Get-Content $settingsFile | ConvertFrom-Json | ConvertTo-Hashtable
             $settings.ApiSettings.Mode = $Config.InstallType
             $settings.ApiSettings.Engine = $Config.engine
-            $settings.ApiSettings.Features=$Config.WebApiFeatures.Features
-            $settings.ApiSettings.BearerTokenTimeoutMinutes=$Config.WebApiFeatures.BearerTokenTimeoutMinutes
-            $settings.ApiSettings.ExcludedExtensions=$Config.WebApiFeatures.ExcludedExtensions
-            $EmptyHashTable=@{}
-            $mergedSettings = Merge-Hashtables $settings, $EmptyHashTable
-            New-JsonFile $settingsFile $mergedSettings -Overwrite
+            If ( $Config.WebApiFeatures.Features -ne $Null -and $Config.WebApiFeatures.Features -ne '') {
+                foreach ($feature in $Config.WebApiFeatures.Features) {
+                      foreach ($defaultfeature in $settings.ApiSettings.Features) {
+                         If ( $feature.Name -eq $defaultfeature.Name) {
+                             $defaultfeature.IsEnabled =$feature.IsEnabled
+                         }
+                      }
+                }
+            }
+           $settings.ApiSettings.BearerTokenTimeoutMinutes=$Config.WebApiFeatures.BearerTokenTimeoutMinutes
+           $settings.ApiSettings.ExcludedExtensions=$Config.WebApiFeatures.ExcludedExtensions      
```

Comment:
```suggestion
            If ($Config.WebApiFeatures.Features -ne $Null -and $Config.WebApiFeatures.Features -ne '') {
                foreach ($feature in $Config.WebApiFeatures.Features) {
                    foreach ($defaultfeature in $settings.ApiSettings.Features) {
                        If ( $feature.Name -eq $defaultfeature.Name) {
                            $defaultfeature.IsEnabled =$feature.IsEnabled
                        }
                    }
                }
            }
           $settings.ApiSettings.BearerTokenTimeoutMinutes=$Config.WebApiFeatures.BearerTokenTimeoutMinutes
           $settings.ApiSettings.ExcludedExtensions=$Config.WebApiFeatures.ExcludedExtensions
```

Reply:
also can we update the example up above to match what the features array should look like:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/e8f7c159647a92ca706a36a359adbdec092c4748/Scripts/NuGet/EdFi.Installer.WebApi/Install-EdFiOdsWebApi.psm1#L134-L148

Reply:
this docs still need to be updated here